public/js/jibe.js

```diff
@@ -253,6 +256,8 @@ var Jibe = (function (BCSocket, CodeMirror, Replay, showdown, Timestamps, TextFo
       }
     });
 
+    /* Initialize table of contents*/
+
```

Comment:
@ctsmith95 There isn't anything TOC related under this comment... leftover?  Probably remove this.